nl.js

```diff
@@ -0,0 +1,747 @@
+settings.crosshair.thickness Dikte
+settings.crosshair.gap Gat
+settings.crosshair.dot Punt
+settings.crosshair.opacity.use Gebruik Eigen Ondoorzichtigheid
+settings.crosshair.opacity.custom Eigen Ondoorzichtigheid
+settings.hitmarker.header Hitmarker
+settings.hitmarker.color Kleur
+settings.hitmarker.kcolor Killkleur
+settings.hitmarker.thickness Dikte
+settings.hitmarker.opacity Ondoorzichtigheid
+settings.hitmarker.leng Lengte
+settings.hitmarker.spacing Spacing
+settings.hitmarker.animD Animatie Grootte
+settings.hitmarker.animS Animatie Snelheid
+settings.hitmarker.hitm Laat zien
+settings.hitmarker.hitmFad Vervaging Snelheid
+settings.hitmarker.custom Hitmarker Afbeelding
+settings.hitmarker.custom.paste Hitmarker Afbeelding URL
+settings.onKillS.style.0 Standaard
+settings.onKillS.style.1 Verminder Resolutie
+settings.onKillS.style.2 Verminder FOV
+settings.onKillS.style.3 Neem FOV toe
+settings.serverb.header Server Browser
+settings.serverb.oldBrowser Legacy Browser
+settings.serverb.defaultRegionOnly Alleen Standaard Regio
+settings.gameprivacy.header Gameplay Privacy
+settings.gameprivacy.streamer Streamer Modus
+settings.gameprivacy.anon Anonieme Modus
+settings.gameprivacy.hideGameTitle Verstop Game Titel
+settings.gameplay.header Gameplay
+settings.gameplay.sensitivityX X Gevoeligheid
+settings.gameplay.sensitivityY Y Govoeligheid
+settings.gameplay.aimsensX Aim X Gevoeligheid
+settings.gameplay.aimsensY Aim Y Gevoeligheid
+settings.gameplay.challenge Challenge Modus
+settings.gameplay.invert Keer X-axis om
+settings.gameplay.onkillS On Kills
+settings.gameplay.scrollDir Scroll Richting
+settings.controller.header Controller
+settings.controller.sensitivityX X Gevoeligheid
+settings.controller.disableCons Schakel Controllers uit
+settings.controller.sensitivityY Y Gevoeligheid
+settings.controller.aimsensX Aim X Gevoeligheid
+settings.controller.aimsensY Aim Y Gevoeligheid
+settings.controller.deadZoneL Dode Zone (Links)
+settings.controller.deadZoneR Dode Zone (Rechts)-
+settings.controller.triggerThres Trigger Threshold
+settings.controller.vibration Vibratie
+settings.controller.gradSpeed Geleidelijke Snelheid
+settings.controller.invert Keer Y-Axis om
+settings.mobile.header Mobiel
+settings.mobile.sensitivityX X Gevoeligheid
+settings.mobile.sensitivityY Y Gevoeligheid
+settings.mobile.aimsensX Aim X Gevoeligheid
+settings.mobile.aimsensY Aim Y Gevoeligheid
+settings.mobile.vibration Vibratie
+settings.mobile.gradSpeed Geleidelijke Snelheid
+settings.mobile.invert Keer Y-Axis om
+settings.audio.header Audio
+settings.audio.sound Meester Volume
+settings.audio.ambient Omringende Volume
+settings.audio.voice Stem Volume
+settings.audio.weps Wapen Volume
+settings.audio.player Speler Volume
+settings.audio.skin Cosmetica Volume
+settings.audio.ui UI Volume
+settings.audio.asset Asset Volume
+settings.audio.dialogue Actie Volume
+settings.viewmodel.header Bekijk model
+settings.viewmodel.fov Gezichtsveld
```

Comment:
'Gezichtsveld' is the outcome in Google Translate I figured. It should be Zichtveld